introduction.md

```diff
@@ -0,0 +1,9 @@
+Hi. This is Cari. My favorite codeschool path is PHP or Git Real. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.